docs/index.md

```diff
@@ -1,12 +1,16 @@
-## Overview
+# Overview
+
 This guide will walk you through the process of deploying a sample Node.js application in CFT.
 An application will be generated from a template and deployed to a Kubernetes cluster.
 At the end of this tutorial, you will be able to access your application via the HMCTS VPN and will have made changes to it.
 
 ## Prerequisites
-GitHub access is required to complete the steps in this tutorial. See the [onboarding guide](https://hmcts.github.io/onboarding/team/github.html#github) to get setup.
 
-Join the [slack channels](https://github.com/hmcts/golden-path-nodejs/blob/master/docs/index.md#slack-channels)
+Before starting this tutorial, make sure you have:
+
+- An account on HMCTS Azure AD. See the [onboarding guide](https://hmcts.github.io/onboarding/person/#azure-ad-groups) to get set up.
+- Access to the HMCTS GitHub organisation. See the [onboarding guide](https://hmcts.github.io/onboarding/team/github.html#github) to get setup.
+- (Recommended) Join the Slack channels
```

Comment:
could also just move the slack channels into a nested list rather than their own section?
```suggestion
- (Recommended) Join the below Slack channels
```